ProteoformSuiteInternal/ProteoformSuiteInternal.csproj

```diff
@@ -65,7 +65,7 @@
       <HintPath>..\packages\FastMember.Signed.1.3.0\lib\net45\FastMember.Signed.dll</HintPath>
     </Reference>
     <Reference Include="FlashLFQ, Version=1.0.0.0, Culture=neutral, processorArchitecture=AMD64">
-      <HintPath>..\packages\mzLib.1.0.298\lib\net45\FlashLFQ.dll</HintPath>
+      <HintPath>..\packages\FlashLFQ.0.1.100\lib\net461\FlashLFQ.dll</HintPath>
```

Comment:
Is this change in line with what is coming down the pipe from mzLib?

Reply:
Yes, I think I have the wrong version for mzlib. Maybe just neglect it, thanks!

Reply:
No worries. I changed this and a bunch of other package listings. I'll commit that after merging this.